docs/src/componentDocs/GettingStarted/Environment.mdx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Divider } from '../../layout/Divider';
+import { CodeBlock } from '../../shared';
+
+{/* KEYWORDS: developer develop environment guide tutorial */}
+
+# Setting up Your Development Environment
+
+Getting started is often the most frustrating part of application development. Fortunately, many of the modern JavaScript frameworks have made great strides in providing tools to get you up and running as quickly as possible. Below, we walk through the basics of getting your system set up to develop.
+
+## Integrated Development Environment (IDE)
+
+There are a bunch of different IDEs on the market, and in general you should be able to use any of these that you like and are comfortable with. If you don't have a personal preference, we recommend trying [Visual Studio Code](https://code.visualstudio.com) - it is quite excellent and comes with a variety of tools to make your development experience seamless. And it's free!
+
+## Git
+
+Chances are, you probably want to keep your application in source control. Git is one of the big players in this space (it's also the backbone for BitBucket).
+
+You may already have git installed. To check, run:
+
+<CodeBlock code={`git --version`} language={'sh'} />
+
+in a command prompt. If you don't have git installed, you can download it from the [Git Website](http://git-scm.com/downloads).
+
+If you are behind a proxy, you'll also need to set up the git proxies as well:
+
+<CodeBlock
```

Comment:
Plese fix indentation.

Reply:
@ektaghag-eaton  fixing indentation is leading to misalignment in code snippet example
<img width="837" alt="Screenshot 2024-12-13 at 11 49 47 AM" src="https://github.com/user-attachments/assets/081ffb7c-7881-4719-925a-8fc225dffe9c" />